src/Snackbar/index.scss

```diff
@@ -0,0 +1,41 @@
+.nds-snackbar {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: wrap;
+  padding: var(--space-m);
+  gap: var(--space-m);
+  min-height: rem(64px);
+  border-radius: rem(24px);
+  box-shadow: var(--elevation-high);
+  width: max-content;
+  min-width: 330px;
+  max-width: 80vw;
+  margin: 0 auto;
+
+  @include atMediaUp("m") {
+    flex-wrap: nowrap;
+    border-radius: rem(56px);
+    padding: var(--space-s) var(--space-m);
+  }
+}
+
+.nds-snackbar-divider,
+.nds-snackbar-buttonGroup {
+  flex-basis: 100%;
+  @include atMediaUp("m") {
+    flex-basis: unset;
+  }
+}
```

Comment:
`flex-basis` is used to force a flex item to wrap to the next line. The basis is relative to the parent, so `100%` does the trick.

This is how we make sure dividers and button groups get their own line while other flex children remain in row flow